### PR TITLE
Updating texturedCube example to use WGSL shaders

### PR DESCRIPTION
**Pull request by ponitka**
Works with Tint with this change: https://dawn-review.googlesource.com/c/tint/+/28125

### CODE REVIEW

File: src/examples/texturedCube.ts [hunk 1]
```diff
@@ -203,3 +203,41 @@ void main() {
...
+[[location 0]] var<out> fragUV : vec2<f32>;
+[[location 1]] var<out> fragPosition: vec4<f32>;
+
+fn vtx_main() -> void {
+  fragPosition = 0.5 * (position + vec4(1.0));
+  Position = uniforms.modelViewProjectionMatrix * position;
+  fragUV = uv;
+  return;
+}
+entry_point vertex as "main" = vtx_main;
```
Comment (kainino0x) [inline src/examples/texturedCube.ts:227]: nit: if `vtx_main` were named `main` this could just be `entry_Point vertex = main;`

**Review by kainino0x — APPROVED** [review]
LGTM but I guess we can wait on the spec before landing.

File: src/examples/texturedCube.ts [hunk 2]
```diff
@@ -203,3 +203,41 @@ void main() {
...
+`,
+  fragment: `
+[[binding = 1, set = 0]] var<uniform> mySampler: sampler;
+[[binding = 2, set = 0]] var<uniform> myTexture: texture_sampled_2d<f32>;
```
Comment (kainino0x) [inline src/examples/texturedCube.ts:231]: Just realized these have `=` while the above ones don't. Pretty sure there should be no `=`. (Syntax is probably going to change soon anyway though.)

**Comment by ponitka** [comment]
> Works with Tint with this change: https://dawn-review.googlesource.com/c/tint/+/28125

Whoops, that was because the sample was still running GLSL shaders.

**Comment by austinEng** [comment]
Thanks for the contribution! :) I ended up cherry-picking your commits and merging them in #60 with a bunch of other WGSL changes.

**Pull request closed without merging**